src/index.css

```diff
@@ -14,6 +14,12 @@
 	--color-border: hsla(220, 13%, 32%, 1);
 	--color-error: var(--color-red);
 	--color-text: var(--color-white);
+
+	--color-branding: #5a23b3;
+	--btn-confirm: var(--color-branding);
+	--btn-confirm-hover: #3f0f7f;
+	--btn-danger: var(--color-error);
+	--btn-danger-hover: #a80000;
```

Comment:
🔍 some of these `--btn` variables could be renamed to suit their use case, IMO. 

For instance: what is `--btn-confirm`? Is it a background color? A text color? A font-size? We know right now that it's a color, but if `--btn-confirm` is referenced in a file far away from here, we have to do some excavation to come back here and understand what the value actually is.

Reply:
fair. I am going to better name these. I really enjoy the naming nits. They all make sense after I look at the original names again with your perspective.

Reply:
I pushed some edits to the naming. Let me know if I understood your suggestion correctly.